frontend/public/components/routes.tsx

```diff
@@ -2,10 +2,13 @@ import * as _ from 'lodash-es';
 import * as React from 'react';
 
 import { ColHead, DetailsPage, List, ListHeader, ListPage, ResourceRow } from './factory';
-import { Cog, ResourceCog, detailsPage, navFactory, ResourceLink, ResourceSummary } from './utils';
+import { Cog, CopyToClipboard, ResourceCog, detailsPage, navFactory, ResourceLink, ResourceSummary } from './utils';
+import { MaskedData } from './configmap-and-secret-data';
+
 import { registerTemplate } from '../yaml-templates';
 // eslint-disable-next-line no-unused-vars
 import {K8sResourceKind, K8sResourceKindReference} from '../module/k8s';
+import {SafetyFirst} from './safety-first';
```

Comment:
nit: I'd change this lines 10-11 to have spaces inside the curly braces to use a consistent style as above.